modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -229,6 +236,34 @@ export default class Ecdsa {
     return shares;
   }
 
+  /**
+   * Derives a child common keychain from common keychain
+   *
+   * @param commonKeychain - common keychain as a hex string
+   * @param path - bip32 path
+   * @return {string} derived common keychain as a hex string
+   */
+  deriveUnhardened(commonKeychain: string, path: string): string {
+    if (Ecdsa.hdTree === undefined) {
+      throw new Error("Can't derive key without HDTree implementation");
+    }
+
+    const keychain = Buffer.from(commonKeychain, 'hex');
+
+    const derivedPublicKeychain = Ecdsa.hdTree.publicDerive(
+      {
+        pk: bigIntFromBufferLE(keychain.slice(0, 32)),
```

Comment:
I suppose for Ed25519, the library we use makes of Little Endian for its underlying operations internally and I suppose secp256k1 uses Big Endian, we probably should use bigIntFromBufferBE for this also. @johnoliverdriscoll kindly correct me if I'm wrong